modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/PanelTag.java

```diff
@@ -110,7 +110,7 @@ protected Map<String, Object> prepareProps(Map<String, Object> props) {
 		props.put("collapseClassNames", _collapseClassNames);
 		props.put("displayTitle", _displayTitle);
 		props.put("displayType", _displayType);
-		props.put("expanded", _expanded);
+		props.put("defaultExpanded", _defaultExpanded);
```

Comment:
I think it would make sense, in this case, to leave `expanded` as the tag attribute and just set here `props.put("defaultExpanded", _expanded);`

The fact that there is a `defaultExpanded` and `expanded` in the Clay component is not related to any business logic. It's a quirk of React, setting the same thing, but in different contexts (controlled vs uncontrolled). From external perspective on somebody using the tag, `defaultExpanded` might be confusing. Somebody could think: what is the meaning of this flag being default? One day we could internally migrate from React to something else, like we moved away from Soy, so tag API should ideally not have specifics to it.

It is a good practice to mirror API of underlying Clay component, but I'd argue that we should completely mirror _capabilities_ of Clay API, and not necessarily the exact props naming.

@dsanz With this change, there would be no need for semver update you mentioned in https://github.com/liferay-frontend/liferay-portal/pull/3507#issuecomment-1612662389